### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -L crate,flor,nd,noice`

### CODE REVIEW

File: apps/game/src/debug/objects.rs [hunk 1]
```diff
@@ -132,7 +132,7 @@ impl ObjectDebugger {
                 let mut f = fs_err::File::create(&path).unwrap();
                 writeln!(f, "{}", json_string).unwrap();
             }
-            // Don't wait for the command to complet.
```
Comment (Robinlovelace) [inline apps/game/src/debug/objects.rs:135]: :+1:

File: headless/examples/go_client.go [hunk 2]
```diff
@@ -29,8 +29,8 @@ var (
 	cityName        = flag.String("city", "us/seattle", "city of the map to simulate")
 	mapName         = flag.String("map", "montlake", "map name to simulate")
 	hoursToSimulate = flag.Int("hours", 24, "number of hours to simulate")
-	comparePct1     = flag.Int64("cmp1", -1, "the baseline percentage for indvidual comparison")
```
Comment (Robinlovelace) [inline headless/examples/go_client.go:32]: Woa this is user-facing by the looks. Good catch!

**Review by Robinlovelace — APPROVED** [review]
This looks great to me, big thumbs up from here in Leeds!

**Comment by Robinlovelace** [comment]
> Found via `codespell -q 3 -L crate,flor,nd,noice`

Out of interest does that also make the changes, or did you make them manually? Impressive either way!

File: apps/ltn/src/select_boundary.rs [hunk 3]
```diff
@@ -154,7 +154,7 @@ impl SelectBoundary {
     }
 
     // Ok(Some(x)) means the current neighbourhood was destroyed, and the caller should switch to
-    // focusing on a different neigbhorhood
+    // focusing on a different neighborhood
```
Comment (dabreegster) [inline apps/ltn/src/select_boundary.rs:157]: Oof, I'm still not consistent with the American/UK spelling. I'll make another pass over this later

**Review by dabreegster — APPROVED** [review]
Awesome, thank you!

**Comment by luzpaz** [comment]
> > Found via `codespell -q 3 -L crate,flor,nd,noice`
> 
> Out of interest does that also make the changes, or did you make them manually? Impressive either way!

You can add the `-i` flag for interactive mode (my preference). You can use the `-w` flag to automatically make the changes.

**Pull request merged**